core/src/test/java/org/apache/iceberg/rest/TestRESTCatalog.java

```diff
@@ -1203,7 +1203,7 @@ public void testTableAuth(
       Map<String, String> expectedContextHeaders,
       Map<String, String> expectedTableHeaders,
       String oauth2ServerUri) {
-    TableIdentifier ident = TableIdentifier.of("ns", "table");
+    TableIdentifier ident = TableIdentifier.of("ns", "tbl");
```

Comment:
if we're introducing `TBL` then we should replace the local `ident` with `TBL`

Reply:
Done